exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -203,7 +196,7 @@ public bool wrongAnswer()
         }
 
 
-        private bool didPlayerWin()
+        private bool DidPlayerWin()
         {
             return !(purses[currentPlayer] == 6);
         }
```

Comment:
Good job on capitalising all methods - I assume this is standard C Sharp practise.